### PR TITLE
Chan: Use `axel` instead of `aria2c`, and async `wget`

### PR DESCRIPTION
**Pull request by wizard-28**
# Purpose
This PR makes use of `axel` to *axel*elerate downloads, instead of `aria2c` as it proved slower and doesn't have a progress bar.

## Progress Report
- [x] Use `axel` in `install-local.sh`
- [x] ~Use `axel` to download `pacscripts`~ (async `wget` is faster for small files)
- [x] Use parallel downloads in `update.sh`
- [x] Use parallel downloads in the installer

### CODE REVIEW

**Review by Henryws — APPROVED** [review]
👍🏻

**Review by NarutoXY — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -104,15 +153,18 @@ mkdir -p $STGDIR
...
 rm -f $STGDIR/repo/pacstallrepo.txt > /dev/null
 touch $STGDIR/repo/pacstallrepo.txt
 echo 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master' > $STGDIR/repo/pacstallrepo.txt
+
 fancy_message info "Pulling scripts from GitHub "
 for i in {add-repo.sh,search.sh,download.sh,install-local.sh,upgrade.sh,remove.sh,update.sh,query-info.sh}; do 
-	wget -q --show-progress -N https://raw.githubusercontent.com/pacstall/pacstall/master/misc/scripts/"$i" -P $STGDIR/scripts
-	chmod +x $STGDIR/scripts/*
-done &
+	wget -q --show-progress -N https://raw.githubusercontent.com/pacstall/pacstall/master/misc/scripts/"$i" -P $STGDIR/scripts &
```
Comment (D-Brox) [inline install.sh:165]: Missing `sudo`

File: install.sh [hunk 2]
```diff
@@ -126,16 +178,16 @@ echo ""
...
-sudo wget -q --show-progress --progress=bar:force -O /bin/pacstall https://raw.githubusercontent.com/pacstall/pacstall/master/pacstall
-sudo chmod +x /bin/pacstall
-
-sudo mkdir -p /usr/share/man/man8/
-wget -q --show-progress --progress=bar:force -O /usr/share/man/man8/pacstall.8.gz https://raw.githubusercontent.com/pacstall/pacstall/master/misc/pacstall.8.gz
-
-sudo mkdir -p /usr/share/bash-completion/completions
-sudo wget -q --show-progress --progress=bar:force -O /usr/share/bash-completion/completions/pacstall https://raw.githubusercontent.com/pacstall/pacstall/master/misc/completion/bash
+sudo wget -q --show-progress --progress=bar:force -O /bin/pacstall https://raw.githubusercontent.com/pacstall/pacstall/master/pacstall &
+wget -q --show-progress --progress=bar:force -O /usr/share/man/man8/pacstall.8.gz https://raw.githubusercontent.com/pacstall/pacstall/master/misc/pacstall.8.gz &
```
Comment (D-Brox) [inline install.sh:182]: Missing `sudo`

File: pacstall [hunk 3]
```diff
@@ -181,6 +181,15 @@ function fancy_message() {
 	esac
 }
 
+# use axel if avaliable
+function download {
+	fancy_message info "$2"
+	if command -v axel > /dev/null; then
+		axel -n $(($(nproc) + 5)) -ao "${1##*/}" "$1"
```
Comment (D-Brox) [inline pacstall:188]: If `wget` uses `sudo`, shouldn't `axel` use too?

Comment (D-Brox) [reply pacstall:188]: Or neither

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (wizard-28) [reply install.sh:165]: The installer's snippet is running with sudo already, like i said, I'll fix this whole mess of `install.sh` in another PR.

> sudo bash -c "$(curl -fsSL https://git.io/JsADh || wget -q https://git.io/JsADh -O -)"

File: install.sh (hunk 2 above)
Comment (wizard-28) [reply install.sh:182]: The installer's snippet is running with sudo already, like i said, I'll fix this whole mess of `install.sh` in another PR.

> sudo bash -c "$(curl -fsSL https://git.io/JsADh || wget -q https://git.io/JsADh -O -)"

**Review by D-Brox — APPROVED** [review]

**Pull request merged**